apps/site-app/src/containers/MainNavigation/MainNavigationMobile/index.tsx

```diff
@@ -45,7 +45,7 @@ export const MainNavigationMobile = () => {
                   </Menu.Item>
                   <Menu.Item>
                     <Icon name="content" onClick={()=>setModalVisible(true)} />
-                      <ModalOperation isOpen={modalVisible}>
+                      <ModalOperation isOpen={modalVisible} closeIcon={true} closeIconFct={()=>setModalVisible(false)}>
```

Comment:
just curious - could we merge `closeIcon` into `closeIconFct`?  Can we ever have one without the other?

Reply:
Done